frontend/src/Project/EditControls/CellTypeControls/CellTypeUI/CellGrid.js

```diff
@@ -1,17 +1,32 @@
 import { Grid } from '@mui/material';
 import Chip from '@mui/material/Chip';
 import FormLabel from '@mui/material/FormLabel';
+import { useSelector } from '@xstate/react';
+import equal from 'fast-deep-equal';
 import { useState } from 'react';
+import { useLayers, useRaw } from '../../../ProjectContext';
+import { hexToRgb } from '../../../service/labels/cellTypesMachine';
```

Comment:
Just double-checking that we don't anticipate any issues with e.g. circular dependencies or anything like that. I'm not familiar with the javascript import system so it may be a moot point - just thought of it due to the depth of the relative imports here.

Reply:
I ran [Madge](https://github.com/pahen/madge) and it seems that we don't have any circular dependencies.